### DISCUSSION TITLE
Reserve Margin - NREL ReEDS

### DISCUSSION THREAD

**Discussion by PriyankaKushwaha940** [NREL/ReEDS-2.0#165] [Q&A: Execution]
Hi,

To calculate the reserve margin on a country or state level, which files (input/output) need to be considered for the data? The files that are close are mentioned along with each line item, please confirm the source

CF (capacity factors, ELCC, technology-wise, year on year) - cfout
Peak demand - peakall
Existing and added capacity - cap_new_ann, cap, cap_firm, cap_firm_iter (not sure which to consider)
Any file/source that can give a better picture like which input files and output files have what kind of information would be super helpful.

Thanks

**Comment by wesleyjcole** [comment]
After the initial years are past, it is unlikely that the model will result in a higher reserve margin than what is required (i.e., it typically builds to the required reserve margin and no more).  This means that the planning reserve margin for a given region is likely to be equal to the input planning reserve margin (inputs/reserves/prm_annual.csv). See https://www.sciencedirect.com/science/article/pii/S0301421518306797 for an example study using ReEDS that looks at the planning reserve margin.

If you want to calculate the planning reserve margindirectly from the outputs, you can use cap_firm and the peak demand.  Cap_firm is the capacity that counts toward the planning reserve margin, so the ratio of cap_firm to peak demand will be the planning reserve margin.

The other relevant output is captrade.csv which shows how much firm capacity was traded from one region to another.  So a region might not have enough firm capacity within the region for the planning reserve, but they have a lot of transmission and can therefore import the firm capacity needed.

Finally, I'll note that there is a lot of development underway at NREL to change how we handle planning reserves in the model, so the 2023 and later versions will include other options besides the traditional planning reserve for maintaining resource adequacye.

**Comment by PriyankaKushwaha940** [comment]
thanks, looking forward to the updates in the new release.
